### PR TITLE
Use cisagov/session-manager-tf-module to configure SSM Session Manager

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request makes use of [cisagov/session-manager-tf-module](https://github.com/cisagov/session-manager-tf-module) to set up access for and configure SSM Session Manager.

## 💭 Motivation and context ##

We want to:
- Stop using ssh tunneling over SSM
- Start logging SSM sessions
- DRY out some repeated code

## 🧪 Testing ##

I successfully applied these changes to the env6 environment in our staging COOL, and afterwards I verified that I was able to create SSM sessions.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Looks great! 🌱

**Comment by jsf9k** [comment]
When #181 is approved and merged, and this PR is rebased, the scripts from #181 must be modified to allow for the SSM Session Manager resources to be correctly created and destroyed.

File: provisionssmsessionmanager_policy_attachment.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ------------------------------------------------------------------------------
+# Attach to the ProvisionAccount role the IAM policy that allows
+# provisioning of the SSM Document resource and set up SSM session
+# logging in this account.
```
Comment (mcdonnnj) [inline provisionssmsessionmanager_policy_attachment.tf]:
```suggestion
# Attach to the ProvisionAccount role the IAM policy that allows all
# of the permissions necessary to provision the SSM Document resource
# and set up SSM session logging in this account.
```

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ Just the same comment suggestion.

File: provisionssmsessionmanager_policy_attachment.tf (hunk 1 above)
Comment (jsf9k) [reply provisionssmsessionmanager_policy_attachment.tf]: Please see commit 8c6e5c1.

**Pull request merged**